<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatabaseDialog</class>
 <widget class="QDialog" name="DatabaseDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QLabel" name="HintLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>311</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Please fill in the information of your database</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>99</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="HostNameLabel">
      <property name="text">
       <string>Host name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="DatabaseNameLabel">
      <property name="text">
       <string>Database name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="PortLabel">
      <property name="text">
       <string>Port </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="UserNameLabel">
      <property name="text">
       <string>User name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="PasswordLabel">
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>181</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="HostNameEdit">
      <property name="focusPolicy">
       <enum>Qt::WheelFocus</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="DatabaseNameEdit">
      <property name="focusPolicy">
       <enum>Qt::WheelFocus</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="PortEdit">
      <property name="focusPolicy">
       <enum>Qt::WheelFocus</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="UserNameEdit">
      <property name="focusPolicy">
       <enum>Qt::WheelFocus</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="PasswordEdit">
      <property name="focusPolicy">
       <enum>Qt::WheelFocus</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>240</y>
     <width>167</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="OKButton">
      <property name="focusPolicy">
       <enum>Qt::TabFocus</enum>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="SkipButton">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
